data/ecosystems/a/arbitrum.toml

```diff
@@ -22,6 +22,7 @@ sub_ecosystems = [
   "GMX",
   "Gravita Protocol",
   "Highstreet Market",
+  "iCat Game",
```

Comment:
Do you mean modify the file like this, sir?

Reply:
yep, thx